### PR TITLE
feat: Keep database credentials in kubernetes secrets

### PR DESCRIPTION
**Pull request by ppawlowski**
## Description

This PR introduces following features:
* move sensitive data to kubernetes secret
* keep database connection parameters in single configuration block
* update README to match proposed changes

## Related Issue(s)

#283 
#268 

## Checklist

 - [x] I have read the [contribution guidelines](https://github.com/FlowFuse/flowfuse/blob/main/CONTRIBUTING.md)
 - [ ] Suitable unit/system level tests have been added and they pass 
 - [ ] Documentation has been updated
    - [ ] Upgrade instructions
    - [ ] Configuration details
    - [ ] Concepts
 - [ ] Changes `flowforge.yml`?
    - [ ] Issue/PR raised on `FlowFuse/helm` to update ConfigMap Template
    - [ ] Issue/PR raised on `FlowFuse/CloudProject` to update values for Staging/Production

## Labels

 - [ ] Backport needed? -> add the `backport` label
 - [ ] Includes a DB migration? -> add the `area:migration` label

### CODE REVIEW

File: helm/flowforge/README.md [hunk 1]
```diff
@@ -180,3 +174,12 @@ editors:
...
 ```
+
+ ### Postgresql
+ - `postgresql.host` - the hostname of an external PostgreSQL database (default not set)
+ - `postgresql.port` - the port of an external PostgreSQL database (default `5432`)
+ - `postgresql.ssl` - sets the connection to the database to use SSL/TLS (default `false`)
+ - `postgresql.auth.username` - the username to use to connect to the database (default `forge`)
+ - `postgresql.auth.password` - the password to use to connect to the database (default `Zai1Wied`)
+ - `postgresql.auth.database` - the database to use (default `flowforge`)
+ - `postgresql.auth.postgresPassword` - the password to use for the postgres user (default `Moomiet0`)
```
Comment (hardillb) [inline helm/flowforge/README.md:185]: The defaults are no longer the case as they have been removed from the `values.yml`

Comment (hardillb) [reply helm/flowforge/README.md:185]: Ignore previous, they are still set

**Comment by hardillb** [comment]
Fresh install looks to work. Will try an upgrade in next

**Comment by hardillb** [comment]
OK, this all looks to work well.

My only question is should we allow the user to specify an existing secret that holds the postgresql passwords?

**Review by hardillb — APPROVED** [review]
I'm approving, but I think we will need another small note in the k8s upgrading to point out the removal of the forge.postgres.* fields incase anybody has changed them.

**Comment by ppawlowski** [comment]
> OK, this all looks to work well.
> 
> My only question is should we allow the user to specify an existing secret that holds the postgresql passwords?

External secret feature will be implemented separately: https://github.com/FlowFuse/helm/issues/290

**Comment by hardillb** [comment]
I will resolve the README.md conflicts and merge this as we've just had a customer ask about this.

**Pull request merged**